.github/workflows/ci.yml

```diff
@@ -60,10 +60,3 @@ jobs:
           pattern: 'packages/**/dist/*.{js,cjs,mjs}'
           exclude: '{**/*.map,**/node_modules/**}'
           repo-token: ${{ secrets.GITHUB_TOKEN }}
-      - if: matrix.command == 'build'
-        uses: transferwise/actions-next-bundle-analyzer@v2
-        with:
-          working-directory: ./websites/visualization
-          comment-strategy: 'always'
-          create-issue: false
-          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
https://github.com/toss/suspensive/pull/1152/files How about removing this too?

Reply:
https://github.com/toss/suspensive/pull/1159/commits/0589614522d48e828b256a6ce530ed4eaa639128